WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BottomSheetNoticeView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
```

Comment:
A comment about what this view is doing would be nice :)

Reply:
Fixed! acd012e9d40236ed00c1738c0aec89ecc21ad49e